### ISSUE TITLE
Can't add price with cents/fractions

### ISSUE THREAD

**Issue by chemicalkosek** [medusajs/admin#630]
Steps to reproduce

1. Edit price of a variant
2. Type in fractional price, in example 19,50
3. After saving see the price rounded down to 19

Price is saved in database as 1900, should be 1950.

**Comment by olivermrbl** [comment]
I can't seem to reproduce this issue. Can I get you to try again with latest admin release?

**Comment by chemicalkosek** [comment]
@olivermrbl Could this be it?
https://github.com/medusajs/admin/blob/a33ed20214297ffdbd2383f809dddd4870f5dad9/src/domain/products/details/variants/variant-editor.tsx#L92

**Comment by chemicalkosek** [comment]
@olivermrbl I have found the culprit of it. I have put a console.log here
https://github.com/medusajs/admin/blob/master/src/domain/products/product-form/sections/prices.tsx#L69
```javascript
                      <CurrencyInput.AmountInput
                              label="Amount"
                              onChange={(amount) =>
                                console.log({ amount }) ||
                                onChange({ ...value, amount })
                              }
                              amount={value?.amount}
                            />
```
And I can see that if I add anything past the `,` it ingnores it in the onChange
Typing `50`      => logs `5000`
Typing `50,50` => logs `5000`

That's because the amount inputs ignores the comma as decimal seperator.
I bet you have in Denmak dot as the decimal seperator
To test this with comma you might want to simulate location in devtools.
Ctrl+Shift+P => Show sensors
Select Berlin from predefined locations as it has comma as decimal seperator too.
https://www.aleksandrhovhannisyan.com/blog/changing-locale-in-chrome-with-dev-tools/

**Comment by kasperkristensen** [comment]
Hi @chemicalkosek,

Thanks for reporting the issue, and sorry for being so slow to act on it. I've just confirmed that this is the case, and will push a fix later today.

We actually also use commas as the decimal separator, so it's a bit strange that we haven't encountered it ourselves, but we must all have our locale set to the US 😅

**Comment by kasperkristensen** [comment]
Fixed with #698, and working in the `development` branch
